OpenRA.Game/Server/ProtocolVersion.cs

```diff
@@ -25,6 +25,8 @@ public static class ProtocolVersion
 		// Order types are:
 		// - 0x65: World sync hash:
 		//   - Int32 containing the sync hash value
+		// - 0x77: Win state report:
```

Comment:
Seems a bit weird to call this "win state" when the bits actually track loss.

Reply:
Renamed to "Defeat state" (`0xDF`)